src/data/equiv/ordered_ring.lean

```diff
@@ -0,0 +1,169 @@
+/-
+Copyright (c) 2021 Alex J. Best. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Alex J. Best
+-/
+import data.equiv.ring
+import algebra.ordered_ring_hom
+
+/-!
+# Ordered ring equivalences
+
+archimedean, and define equivalences between these fields. We also construct the natural map from a
```

Comment:
Looks like the text got cut off here?

Reply:
Oops thanks, I'm still working on this right now, so I'll mark as needs review when it's good to go again.